documentation/docs/scenarios/ui5-sap-cp/Readme.md

```diff
@@ -45,42 +38,45 @@ Following the convention for pipeline definitions, use a `Jenkinsfile` which res
 fioriOnCloudPlatformPipeline script:this
 ```
 
-`fioriOnCloudPlatform` is a so called _scenario step_ that wraps the [mtaBuild](https://sap.github.io/jenkins-library/steps/mtaBuild/) and the [neoDeploy](https://sap.github.io/jenkins-library/steps/neoDeploy/) steps.
+[fioriOnCloudPlatformPipeline](https://sap.github.io/jenkins-library/steps/fioriOnCloudPlatformPipeline) is a so called _scenario step_ that wraps mainly the [MTA Build](https://sap.github.io/jenkins-library/steps/mtaBuild/) step and the [SAP Cloud Platform Deployment](https://sap.github.io/jenkins-library/steps/cloudFoundryDeploy/) step.
 
-### Configuration (`.pipeline/config.yml`)
+### Scenario Configuration
 
-This is a basic configuration example, which is also located in the sources of the project. The configuration corresponds to the steps wrapped in `fioriOnCloudPlatformPipeline`.
+This is a basic configuration example, which also resides in the sources of the project. Edit `.pipeline/config.yml` below the root directory. The configuration corresponds to the steps wrapped in `fioriOnCloudPlatformPipeline`. Refer to the 
+[mtaBuild](https://sap.github.io/jenkins-library/steps/mtaBuild/) and [cloudFoundryDeploy](https://sap.github.io/jenkins-library/steps/cloudFoundryDeploy/) documentation for details.
 
 ```yaml
 steps:
   mtaBuild:
-    buildTarget: 'NEO'
-    platform: 'neo'
+    platform: 'CF'
+  cloudFoundryDeploy:
+    cloudFoundry:
+      apiEndpoint: 'your-cf-endpoint' # default: 'https://api.cf.eu10.hana.ondemand.com'
+      org: 'your-org-id'
+      space: 'your-space-id'
+      credentialsId: 'CF-jenkins-credentials-id'
+```
+
+### Optional environment
+
+Optionally, [fioriOnCloudPlatformPipeline](https://sap.github.io/jenkins-library/steps/fioriOnCloudPlatformPipeline) allows to deploy into  a NEO environment. Refer to the [neoDeploy](https://sap.github.io/jenkins-library/steps/neoDeploy/) documentation for details.
+
+```yaml
+steps:
+  mtaBuild:
+    platform: 'NEO'
   neoDeploy:
     neo:
-      credentialsId: 'NEO_DEPLOY'
+      credentialsId: 'NEO-jenkins-credentials-id'
       account: 'your-account-id'
       host: 'hana.ondemand.com'
 ```
 
-#### Configuration for the MTA Build
-
-| Parameter        | Description    |
-| -----------------|----------------|
-| `buildTarget`    | The target platform to which the mtar can be deployed. Possible values are: `CF`, `NEO`, `XSA` |
-| `mtaJarLocation` | The location of the multi-target application archive builder jar file, including file name and extension. |
```

Comment:
This, especially the `mtaJarLocation` is really outdated and needs to be cleaned up  ...

Reply:
the central documentation isn't updated yet. i removed the complete MTA configuration paragraph including these properties.